### PR TITLE
Update parsing logs with logstash

### PR DESCRIPTION
**Pull request by dedemorton**

### CODE REVIEW

**Comment by dedemorton** [comment]
@suyograo and @acchen97 I've updated the part of the getting started that describes how to parse apache logs. I've made Filebeat the input and added made the steps more iterative. If you want, you can go ahead and review this part now. I'm not going to merge the changes, though, until I've made the other changes that we've discussed.

File: docs/static/advanced-pipeline.asciidoc [hunk 1]
```diff
@@ -4,14 +4,72 @@
...
+To get started, go https://download.elastic.co/demos/logstash/gettingstarted/logstash-tutorial.log.gz[here] to
+download the sample data set used in this example. Unpack the file.
+
+
+[[configuring-filebeat]]
+==== Configuring Filebeat to Send Log Lines to Logstash
+
+Before you create the Logstash pipeline, you'll configure Filebeat to send log lines to Logstash.  https://github.com/elastic/beats/tree/master/filebeat[Filebeat] is a lightweight, resource-friendly tool that
+collects logs from files on the server and forwards these logs to your Logstash instance for processing. Filebeat is 
+designed for reliability and low latency. Filebeat uses the computing resources of the machine hosting the source data,
```
Comment (suyograo) [inline docs/static/advanced-pipeline.asciidoc]: > Filebeat uses the computing resources of the machine hosting the source data

This kind of indicates that FB uses a lot of resource on host machines. How about rephrasing this like "Filebeat is designed to ship logs with a light resource footprint on the host machine"

**Comment by suyograo** [comment]
Minor comment, otherwise LGTM

File: docs/static/advanced-pipeline.asciidoc [hunk 2]
```diff
@@ -4,14 +4,72 @@
...
 --------------------------------------------------------------------------------
+sudo ./filebeat -e -c filebeat.yml -d "publish"
+--------------------------------------------------------------------------------
+
+Filebeat will attempt to connect on port 5403. Until Logstash starts with an active Beats plugin, there
+won’t be any answer on that port, so any messages you see regarding failure to connect on that port are normal for now.
+
+==== Configuring Logstash for Filebeat Input
+
+Next, you create a Logstash configuration pipeline that uses the Beats input plugin to receive
```
Comment (acchen97) [inline docs/static/advanced-pipeline.asciidoc]: I'd also note somewhere that the Beats input can receive data from all the different Beats, not just Filebeat.

**Comment by acchen97** [comment]
LGTM, great work @dedemorton. I really like the iterative way the example is structured.

If possible, can we maybe briefly mention or show a snapshot of the Kibana dashboard that ships with Filebeat here? It's a good way to close the loop and also keep LS pronounced in the stack story. :)

**Comment by suyograo** [comment]
LGTM. nice work!

**Pull request closed without merging**